docs/docs/developers/tutorials/codealong/contract_tutorials/token_contract.md

```diff
@@ -408,7 +408,7 @@ aztec codegen target -o src/artifacts
 
 ### Token Bridge Contract
 
-The [token bridge tutorial](./token_bridge) is a great follow up to this one.
+The [token bridge tutorial](../js_tutorials/token_bridge) is a great follow up to this one.
```

Comment:
Does this need to change in versioned docs as well?

Reply:
Yes, I intend to bring this and any other tutorial fixes to the other versions once complete